lib/views/github-tab-view.js

```diff
@@ -47,10 +47,8 @@ export default class GitHubTabView extends React.Component {
 
     if (this.props.currentRemote.isPresent()) {
       // Single, chosen or unambiguous remote
-      // only supporting GH.com for now, hardcoded values
```

Comment:
We _are_ still supporting dotcom... but the Endpoint stuff does help lay the groundwork for actual Enterprise-readiness.